oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/BanyanDB.java

```diff
@@ -177,4 +177,34 @@ enum IndexType {
          */
         int countersNumber() default 1000;
     }
+
+    /**
+     * Match query is designed for BanyanDB match query with specific analyzer. It is a fuzzy query implementation
+     * powered by analyzer.
+     *
+     * @since 10.1.0
+     */
+    @Target({ElementType.FIELD})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface MatchQuery {
+        AnalyzerType analyzer() default AnalyzerType.ANALYZER_SIMPLE;
```

Comment:
As @hanahmily is adding a more correct analyzer, I think we should consider no default for this annotation.